controller/products.js

```diff
@@ -18,14 +20,22 @@ const sellers = (req, res) => {
 };
 
 const quotes = (req, res) => {
-  console.log(req.query);
-  let priceInfo;
-  let sellerInfo;
   let id = null;
   if (req.query.productId) {
     id = req.query.productId;
   }
 
+  if (id && isNaN(Number(id))) {
+    return res.status(400).send('Bad Request.');
+  }
+
+  if (id && (id < 1 || id > 100)) {
+    return res.status(404).send('Product Not Found.');
+  }
```

Comment:
This is FEC specific. Are you not able to just query your database and send a 404 if nothing is found?

Reply:
While I can, I don't want to handle any logic at the DB level and instead return whatever data is found or not. But to your point, I have made the logic dynamic instead of static check.